common/lib/xmodule/xmodule/html_module.py

```diff
@@ -75,6 +75,15 @@ def student_view(self, _context):
         """
         return Fragment(self.get_html())
 
+    def student_view_data(self, context=None):
+        """
+        Returns a JSON representation of the student_view of this XBlock,
+        retrievable from the Course Block API.
```

Comment:
The docstring should state that because `student_view_data` is not user-specific, the returned HTML may contain `%%USER_ID%%` which (unlike `student_view`) will not yet have been replaced with the student's anonymous ID.